plugin.yml

```diff
@@ -1,6 +1,6 @@
 name: PEXProtection
 author: "RevivalPMMP"
-api: [3.0.0-ALPHA9, 3.0.0-ALPHA10, 3.0.0-ALPHA11]
+api: [3.0.0-ALPHA12, 3.0.0-ALPHA13, 3.0.0, 3.0.1, 3.0.2, 3.0.3, 3.0.4, 3.0.5, 3.1, 4.0.0]
```

Comment:
remove 3.0.5, 3.1, and 4.0.0

Reply:
It should not be necessary to declare anything other than 3.0.0 as the new versioning system will still load the plugin for any 3.0.x version.

Specific numbers in the last place should only be used if there is a change in that version of PMMP that the plugin requires.  This will keep the plugin from loading on versions earlier than the declared but not newer.